ci/prepare-deploy-travis.sh

```diff
@@ -24,13 +20,13 @@ if [[ "$TARGET" == "x86_64-unknown-linux-gnu" && "$TRAVIS_BRANCH" == "stable" ]]
 fi;
 
 # Copy rustup-init to rustup-setup for backwards compatibility
-cp target/$TARGET/release/rustup-init target/$TARGET/release/rustup-setup
+cp target/"$TARGET"/release/rustup-init target/"$TARGET"/release/rustup-setup
 
 # Generate hashes
-if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-    find "target/$TARGET/release/" -maxdepth 1 -type f -exec sh -c 'shasum -a 256 -b "{}" > "{}.sha256"' \;;
+if [ "$TRAVIS_OS_NAME" = "osx" ]; then
+    find target/"$TARGET"/release/ -maxdepth 1 -type f -exec sh -c 'fn="$1"; shasum -a 256 -b "$fn" > "$fn".sha256' sh {} \;
 else
-    find "target/$TARGET/release/" -maxdepth 1 -type f -exec sh -c 'sha256sum -b "{}" > "{}.sha256"' \;;
+    find target/"$TARGET"/release/ -maxdepth 1 -type f -exec sh -c 'fn="$1"; sha256sum -b "$fn" > "$fn".sha256' sh {} \;
```

Comment:
Could you explain this one to me, please?

The previous version looked clean, simple and correct.

Reply:
ShellCheck warns: 
```
find "target/$TARGET/release/" -maxdepth 1 -type f -exec sh -c 'shasum -a 256 -b "{}" > "{}.sha256"' \;;
>>                                                             ^-- SC2156: Injecting filenames is fragile and insecure. Use parameters.
```

Reply:
Is the extra `sh` on that line necessary?  I grok the rest.

Reply:
Got it.  Thanks.
The extra `sh` is set as`$0` which isn't unused.  I understand it to be so that the argument is a `$1`.